tests/plugins/font/manual/fontcommand.md

```diff
@@ -0,0 +1,30 @@
+@bender-tags: 4.14.0, feature, 3306, 3728
```

Comment:
`3306` has been treated as general issue, but as we split it into smaller pieces, maybe only `3728` is just fine?